.github/workflows/integration.yml

```diff
@@ -0,0 +1,120 @@
+# Run secret-dependent integration tests only after /ok-to-test approval
+on:
+  pull_request:
+  repository_dispatch:
+    types: [ok-to-test-command]
+
+name: Integration tests
+
+jobs:
+  integration-trusted:
+    runs-on: ubuntu-latest
+    if: (github.event_name == 'repository_dispatch') || (github.event_name == 'pull_request' && github.event.pull_request.head.repo.full_name == github.repository)
+    steps:
+      - id: debug-outputs
+        run: |
+          echo ${{ github.event_name }}
```

Comment:
Do we need all these echos now?

Reply:
Echo's are needed for debugging. If it works the way we expect it to then I will do another PR to remove them all.

Reply:
sounds good, perfect then!